packages/renderer/src/lib/feedback/SendFeedback.svelte

```diff
@@ -110,17 +116,27 @@ async function sendFeedback(): Promise<void> {
           name="contactInformation"
           id="contactInformation"
           bind:value="{contactInformation}"
-          placeholder="Enter email address, phone number or leave blank for anonymous feedback"
+          placeholder="Enter email address, phone number, or leave blank for anonymous feedback"
```

Comment:
As it's very unlikely that user will give their phone number, can we switch to type=email and update placeholder

Reply:
Updated.